src/main.py

```diff
@@ -442,6 +445,21 @@ def get_arg_parser():
         required=True,
         help="Specify the target patch version for which the package dependency report needs to be generated.",
     )
+    package_release_parser = subparsers.add_parser(
+        "generate-version-release",
+        help="Generates release for each new image version.",
+    )
+    package_release_parser.set_defaults(func=generate_new_version_release)
+    package_release_parser.add_argument(
+        "--target-patch-version",
+        required=True,
+        help="Specify the target patch version for which the release needs to be generated.",
+    )
+    package_release_parser.add_argument(
+        "--image-type",
```

Comment:
Are we doing 2 separate github releases, 1 for each image type?

Reply:
Yes, it was decided in meeting with PM. Reason is in the future we may have release for other platform like `trainium`